CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ Main (unreleased)
 - (_Experimental_) Various changes to the experimental component `database_observability.mysql`:
   - `schema_table`: add support for index expressions (@cristiangreco)
 
+- Update `async-profiler` binaries for `pyroscope.java` to 4.0-87b7b42 (@github-hamza-bouqal)
```

Comment:
Please merge in main to confirm that this is going in the correct part of the changelog.